internal/cli/tables-to-go-cli.go

```diff
@@ -329,3 +311,43 @@ func replaceSpace(r rune) rune {
 	}
 	return r
 }
+
+// FormatColumnName checks for invalid characters and transforms a column name
+// according to the provided settings.
+func formatColumnName(settings *settings.Settings, column, table string) (string, error) {
+
+	// Columns that don't start with a capital letter must have one prefixed
+	// to be public fields
+	var prefix string
+	if settings.IsOutputFormatCamelCase() {
+		prefix = "X"
+	} else {
+		prefix = "X_"
+	}
```

Comment:
Can you move the `prefix` creation more to the place (line 343) where it's actually used? Also using `else` is not necassary, we can stick to our default OutputFormat (camelCase) and rewrite it like

```go
prefix := "X"
if !settings.IsOutputFormatCamelCase() {
	prefix = "X_"
}
```

Maybe for the sake of clean code we can introduce a new method in settings `IsOutputFormatOriginal` to get rid of the negation in the if-statement - but not necessary!

Reply:
Moved the creation in 584dbfa and I changed the logic to avoid the negation:
```go
prefix := "X_"
if settings.IsOutputFormatCamelCase() {
	prefix = "X"
}
```